tealer/teal/teal.py

```diff
@@ -10,6 +10,47 @@ def __init__(self, instructions: List[Instruction], bbs: List[BasicBlock]):
         self._instructions = instructions
         self._bbs = bbs
 
+    @staticmethod
+    def render_instruction(i: Instruction):
+        ins_str = str(i).replace('"', '\"')
+        ins_str = ins_str.replace('>', '&gt;')
+        ins_str = ins_str.replace('<', '&lt;')
+        ins_str = ins_str.replace('&&', '&amp;&amp;')
+        comment_str = "no comment for this line"if i.comment == "" else i.comment
+        comment_str = comment_str.replace('*', '&#42;')
+        comment_str = comment_str.replace('>', '&gt;')
+        comment_str = comment_str.replace('<', '&lt;')
```

Comment:
Is there a reason why we don't do `comment_str = comment_str.replace('&&', '&amp;&amp;')`? (may be nice to add a comment for that)

Reply:
I need to take a closer look, I have probably dropped the relevant line for some reason. Let me get back to you on that later.